hack/make-rules/vet.sh

```diff
@@ -25,8 +25,14 @@ cd "${KUBE_ROOT}"
 
 # This is required before we run govet for the results to be correct.
 # See https://github.com/golang/go/issues/16086 for details.
-make generated_files
-go install ./cmd/...
+# If called directly, exit.
+CALLED_FROM_MAIN_MAKEFILE=${CALLED_FROM_MAIN_MAKEFILE:-""}
+if [ "${CALLED_FROM_MAIN_MAKEFILE}" == "" ]; then
```

Comment:
I might've just done `if [[ -z "${CALLED_FROM_MAIN_MAKEFILE:-}" ]]; then` since it's shorter, but this is fine too.